mix.exs

```diff
@@ -28,15 +28,15 @@ defmodule Kaffe.Mixfile do
   defp deps do
     [
       {:brod, "~> 3.0"},
-      {:ex_doc, "~> 0.14", only: :dev, runtime: false},
+      {:ex_doc, "~> 0.20", only: :dev, runtime: false},
     ]
   end
 
   defp package do
     [
       name: :kaffe,
       licenses: ["MIT License"],
-      maintainers: ["Kevin Lewis", "David Santoso", "Ryan Daigle", "Spreedly"],
+      maintainers: ["Kevin Lewis", "Spreedly", "Joe Peck"],
```

Comment:
🤔 I'm on the fence if we should remove maintainers on an open-source project. Theoretical they could still contribute?

Also, we'll want to publish this release to Hex. Looks like @objectuser was the last to do so. https://hex.pm/packages/kaffe

Reply:
@jknipp Ah, good point. I added them back, since it's open-source.